### PR TITLE
Disable sprite wobble

### PR DESCRIPTION
**Pull request by devxan**
Not attached to any issues from ScratchAddons, but is in LLK/scratch-gui/issues/5471

### Changes

Disables wobble in sprites pane for dragging sprites, costumes and audio files like [this](https://youtu.be/auyMTDUoMvE).

### Reason for changes

Sprite wobbling is a large annoyance that makes moving anything from sprite to sprite nearly impossible with many displays.

### Tests

I have used this addon and it works very well. 

### Possible changes

I might add a setting in the future that plays other animations instead of the rotation and size change that happens in Scratch currently.

### CODE REVIEW

**Comment by RedGuy12** [comment]
add `"versionAdded": "1.24.0"` to manifest

File: addons/disable-sprite-wobble/userstyle.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[class*="sprite-selector_raised"]:hover {
+  -webkit-animation-name: none;
+  animation-name: none;
+  -webkit-transform: none;
+  -ms-transform: none;
+  transform: none;
```
Comment (WorldLanguages) [inline addons/disable-sprite-wobble/userstyle.css:6]: Remove the prefixed properties

Comment (devxan) [reply addons/disable-sprite-wobble/userstyle.css:3]: Done ✅

File: addons/disable-sprite-wobble/addon.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      "link": "https://github.com/cst1229/"
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "userstyle.css",
+      "matches": ["projects"]
+    }
+  ],
+  "tags": ["editor", "recommended"],
```
Comment (WorldLanguages) [inline addons/disable-sprite-wobble/addon.json:20]: Not recommended

Comment (RedGuy12) [reply addons/disable-sprite-wobble/addon.json:20]: like i said

Comment (devxan) [reply addons/disable-sprite-wobble/addon.json]: It's been altered ☑️

File: addons/disable-sprite-wobble/userstyle.css (hunk 1 above)
Comment (CST1229) [reply addons/disable-sprite-wobble/userstyle.css:3]: I copied these properties from scratch btw

Comment (devxan) [reply addons/disable-sprite-wobble/userstyle.css:3]: That's true. But Scratch Addons is only used on Firefox and Chromium browsers, so it shouldn't be much of an issue.

Comment (RedGuy12) [reply addons/disable-sprite-wobble/userstyle.css:3]: Webkit is chromium

**Comment by RIMOPA** [comment]
I make the revert inadvertently, is an error, please ignore  it.

**Pull request merged**